app/components/rating/rating.twig

```diff
@@ -0,0 +1,7 @@
+{% macro default() %}
+  <i class="fa fa-star fa-lg rating"></i>
+  <i class="fa fa-star fa-lg rating"></i>
+  <i class="fa fa-star fa-lg rating"></i>
+  <i class="fa fa-star-half-o fa-lg rating"></i>
+  <i class="fa fa-star-o fa-lg rating"></i>
+{% endmacro %}
```

Comment:
Instead of writing the markup several times we could use the `icons` component in order to keep the source code free from repeating content.